config/302-pac-configmap.yaml

```diff
@@ -42,6 +42,14 @@ data:
   # Add extra IPS (ie: 127.0.0.1) or networks (127.0.0.0/16) separated by commas.
   bitbucket-cloud-additional-source-ip: ""
 
+  # max-keep-runs defines a max number of run that will be kept on the cluster for a Pipeline
+  # if a user has defined annotation on pipelineRun, it's value should be less than or equal to
```

Comment:
```suggestion
  # if a user has defined annotation on pipelineRun, its value should be less than or equal to
```

Reply:
I am confused when i was reading about  this option.
 maybe the value is not named properly?

perhaps `max-keep-run-upper-limit`

Reply:
yeah this is better